bundler/spec/install/gems/native_extensions_spec.rb

```diff
@@ -88,6 +88,62 @@
     expect(out).to eq("true")
   end
 
+  it "installs from git when multiple gems share one repository" do
+    build_repo2 do
+      ["one", "two"].each do |n|
+        build_lib "c_extension_#{n}", "1.0", :path => lib_path("gems/c_extension_#{n}") do |s|
+          s.extensions = ["ext/extconf.rb"]
+          s.write "ext/extconf.rb", <<-E
+            require "mkmf"
+            name = "c_extension_bundle_#{n}"
+            dir_config(name)
+            raise "OMG" unless with_config("c_extension_#{n}") == "#{n}"
+            create_makefile(name)
+          E
+
+          s.write "ext/c_extension_#{n}.c", <<-C
+            #include "ruby.h"
+
+            VALUE c_extension_#{n}_value(VALUE self) {
+              return rb_str_new_cstr("#{n}");
+            }
+
+            void Init_c_extension_bundle_#{n}() {
+              VALUE c_Extension = rb_define_class("CExtension_#{n}", rb_cObject);
+              rb_define_method(c_Extension, "value", c_extension_#{n}_value, 0);
+            }
+          C
+
+          s.write "lib/c_extension_#{n}.rb", <<-C
+            require "c_extension_bundle_#{n}"
+          C
+        end
+      end
+      build_git "gems", :path => lib_path("gems"), :gemspec => false
+    end
+
+    bundle! "config set build.c_extension_one --with-c_extension_one=one"
+    bundle! "config set build.c_extension_two --with-c_extension_two=two"
+
+    # 1st time, require only one gem -- only one of the extensions gets built.
+    install_gemfile! <<-G
+      gem "c_extension_one", :git => #{lib_path("gems").to_s.dump}
+    G
+
+    expect(out).not_to include("extconf.rb failed")
```

Comment:
I know this is probably copied from other specs in this file that do the same, but this line is not necessary, since the `install_gemfile!` will fail earlier in case of an error when building the extension.

Reply:
Yep, that was taken from the other adjacent tests to be consistent with them. I'm fine with removing it since the final expectation can only pass if everything succeeds with the build - should I remove it from all the other tests in that file too?

Reply:
I'm doing that in #3479, so not necessary!